c2corg_ui/static/js/documentediting.js

```diff
@@ -504,6 +504,22 @@ app.DocumentEditingController.prototype.pushToArray = function(object, property,
   app.utils.pushToArray(object, property, value, event);
 };
 
+/**
+ * @param {number} value
+ * @export
+ */
+app.DocumentEditingController.prototype.convertExposureTime = function(value) {
+  if (value === 0) {
+    this['scope']['image']['exposure_time_converted'] = this.alerts.gettext('cannot be 0');
+  } else if (value < 1 && value) {
+    this['scope']['image']['exposure_time_converted'] = '1/' + (1 / value) + ' s';
+  } else if (value >= 1) {
+    this['scope']['image']['exposure_time_converted'] = value + ' s';
+  } else if (!value) {
+    this['scope']['image']['exposure_time_converted'] = '';
```

Comment:
Perhaps you could factorize a bit in order to write ``this['scope']['image']['exposure_time_converted']`` only once.